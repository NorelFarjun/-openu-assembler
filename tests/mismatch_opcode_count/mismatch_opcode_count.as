stop r1
add r3, r3, r4
sub r6
prn #6, r1
jsr
