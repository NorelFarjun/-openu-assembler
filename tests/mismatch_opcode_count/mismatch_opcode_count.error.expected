0: extra operands for instruction 'stop'
1: extra operands for instruction 'add'
2: missing operands for instruction 'sub'
3: extra operands for instruction 'prn'
4: missing operands for instruction 'jsr'
Bad input file - not outputting
